internal/inputs/http.go

```diff
@@ -89,6 +89,17 @@ func RunHTTP(dataStore *[]interface{}, doLoop *bool, yml *load.Config, api load.
 				if api.Pagination.OriginalURL == "" || (api.Pagination.OriginalURL != "" && resp.StatusCode >= 200 && resp.StatusCode <= 299) && addPage {
 					handleJSON(dataStore, body, &resp, doLoop, reqURL, nextLink)
 				}
+			case contentType == "text/xml" || contentType == "application/xml":
+				jsonBody, err := xj.Convert(resp.Body)
+				if err != nil {
+					load.Logrus.WithFields(logrus.Fields{
+						"err": err,
+					}).Errorf("http: URL %v failed to convert XML to Json resp.Body", *reqURL)
```

Comment:
It would be better to use WithError(err) instead of WithFields("err"). It's a bit more "standard".